components/GoogleSignIn/GoogleSignIn.tsx

```diff
@@ -56,7 +59,7 @@ function GoogleSignIn(props: Props) {
         setIsLoggedIn(true)
         api.setGoogle(response.tokenId)
             .then(() => {
-                (window as any).googleAuthObj = response
+                ;(window as any).googleAuthObj = response
```

Comment:
Why is the ; at the start?

Reply:
https://stackoverflow.com/questions/7145514/whats-the-purpose-of-starting-semi-colon-at-beginning-of-javascript

Formatter wants it that way